compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -254,8 +255,21 @@ class ResolverImpl(
 
         val superDescriptor = resolveForOverride(overridee) as? CallableMemberDescriptor ?: return false
         val subDescriptor = resolveForOverride(overrider) as? CallableMemberDescriptor ?: return false
+        val subClassDeclaration = overrider.closestClassDeclaration()?.let {
```

Comment:
Actually I was referring to name it to `subClassDescriptor` as it is the descriptor representation of the closest declaration.

Reply:
whoops sorry didn't notice resolveClassDeclaration returns a descriptor. fixed.